7_ReleaseNotes.txt

```diff
@@ -18,6 +18,13 @@
   - Recurrence fixes
   - Pseudoprogression fixes
 - Generic GUI bug fixes and improvements 
+- Apply mask functionaity from drawing panel added
+- bvec reorientation functionality added from Utilities
+- Headers added to csv features files for multiple applications
+- Fixed application font mismatch on GUI
+- Fixed mask opacity change functionaity from comparison mode
+- Fixes for issues identified during Testathon Feb 2020
```

Comment:
Remove this, as it has got nothing to do with the code and getting pertinent information will involve going to GitHub and searching for relevant tag.

Reply:
removed testathon part. Is it better now?